draft-ietf-ivy-network-inventory-yang.md

```diff
@@ -164,55 +164,58 @@ The YANG data model defined in this document conforms to the Network Management
 
 ## Terminology
 
-  The following terms are defined in {{!RFC7950}} and are not
-  redefined here:
+The following terms are defined in {{!RFC7950}} and are not redefined here:
+- client
+- server
+- augment
+- data model
+- data node
 
-  *  client
+The following terms are defined in {{!RFC6241}} and are not redefined here:
+- configuration data
+- state data
 
-  *  server
+The following terms are defined in {{IANA_YANG}} and are not redefined here:
+- backplane
+- battery
+- container
+- cpu
+- chassis
+- fan
+- module
+- port
+- power supply
+- sensor
+- stack
+- storage device
 
-  *  augment
+> Note that the definition of port component in {{IANA_YANG}} needs to be refined in future version of this document.
 
-  *  data model
+> TBD: Recap the concept of chassis/slot/component/board/... in {{TMF_SD2-20}}.
 
-  *  data node
+Also, the document makes use of the following terms:
 
-  The following terms are defined in {{!RFC6241}} and are not redefined
-  here:
+Network Inventory:
+: A collection of data for network elements and their components managed by a specific management system.
 
-  *  configuration data
+Physical Network Element:
+: An implementation or application specific group of components (e.g., hardware components).
 
-  *  state data
+Network Element:
+: The generalization of the physical network element definition to include other types of inventory objects which can be managed, from an inventory perspective, like physical network elements.
 
-  The terminology for describing YANG data models is found in {{!RFC7950}}.
+Hardware Component:
+: The generalization of the hardware components defined in {{IANA_YANG}} (e.g., backplane, battery, container, cpu, chassis, fan, module, port, power supply, sensor, stack and storage device components).
+: The list of hardware components can be extended in future versions of {{IANA_YANG}}.
```

Comment:
We can't formally directly add those the IANA module. These should be added to IANA-ENTITY-MIB:

```
>    The "iana-hardware" YANG module is intended to reflect the
>    "IANA-ENTITY-MIB" MIB module so that if a new enumeration is added to
>    the "IANAPhysicalClass" textual convention, the same class is added
>    as an identity derived from "ianahw:hardware-class".
```